src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/DxAddressCreationRequest.java

```diff
@@ -2,21 +2,23 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Getter;
 
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Pattern;
-
-import lombok.Builder;
-import lombok.Getter;
+import javax.validation.constraints.Size;
 
 @Getter
 @Builder(builderMethodName = "dxAddressCreationRequest")
 public class DxAddressCreationRequest {
 
     @NotNull
     @Pattern(regexp = "^(?:DX|NI) [0-9]{10}+$")
+    @Size(max = 13)

```

Comment:
please remove since we have used separate validator for this, also for dxExchange

Reply:
resolved